suites/test_module_content.py

```diff
@@ -12,6 +12,8 @@
 import suites.test_module_edit_publish as test_module_edit_publish
 import os
 
+PROXIES = {'http': 'http://squid.corp.redhat.com:3128', 'https': 'http://squid.corp.redhat.com:3128'}
```

Comment:
@pbadak please move the proxy config to config.ini, actual values don't go here!